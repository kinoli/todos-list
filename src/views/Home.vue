<template>
  <Listing />
</template>

<script>
import Listing from '../components/Listing'

export default {
  components: {
    Listing
  }
}
</script>
