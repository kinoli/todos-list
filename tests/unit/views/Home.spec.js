import { shallowMount } from '@vue/test-utils'
import Home from '@/views/Home.vue'

describe('Home.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Home)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
