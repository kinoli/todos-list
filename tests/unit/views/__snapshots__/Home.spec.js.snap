// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue renders correctly 1`] = `<listing-stub></listing-stub>`;
